config/crd/serving.kserve.io_clusterservingruntimes.yaml

```diff
@@ -653,6 +653,8 @@ spec:
                 type: string
               httpDataEndpoint:
                 type: string
+              multiModel:
+                type: boolean
```

Comment:
Can it be "colocate"? Not sure -we understand single/multi-model literature well, but at the basic level it's a decision to colocate or not?